checkov/terraform/utils.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+from typing import Optional, Any
+
+
+class TFModule:
```

Comment:
`utils.py` is not a great name.
What about creating a new directory (modules/module_objects) with a file for each class?

Reply:
Moved to `modules/module_objects.py` i think that file per class is too much here.